TileStache/Goodies/VecTiles/sort.py

```diff
@@ -76,8 +77,16 @@ def landuse(features, zoom):
     return _sort_by_area_then_id(features)
 
 
+def _place_key_desc(feature):
+    sort_key = _by_population(feature), _by_area(feature)
+    return sort_key
+
+
 def places(features, zoom):
-    return _sort_by_scalerank_then_population(features)
+    features.sort(key=_place_key_desc, reverse=True)
+    features.sort(key=_by_scalerank)
```

Comment:
This will need to be `min_zoom` once data changes (done in https://github.com/whosonfirst/whosonfirst-data/issues/146) are integrated here in vector-datasource.